test/test_helper.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""
+Copyright 2016 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import os
+import mbed_lstools
+
+
+# pylint: disable=too-few-public-methods
+class Helper(object):
+    """
+    Helper class for all tests
+    """
+    def __init__(self, platform_name, allowed_files):
+        self.platform_name = platform_name
+        self.allowed_files = []
+        self.allowed_files.extend(allowed_files)
+
+    def clear(self):
+        """
+        clear Files should not be put into mount point
+        """
+        if self.platform_name:
+            targets = self._get_targets()
+
+            # Clear files other than allowed
+            for target in targets:
+                mount_point = target.get('mount_point')
+                if mount_point:
+                    for bad_file in os.listdir(mount_point):
+                        if os.path.isfile(bad_file) and bad_file not in self.allowed_files:
+                            os.remove(os.path.join(mount_point, bad_file))
```

Comment:
Should there be some handling for possible errors, like missing permissions to handle the files?

Reply:
setup/teardown will fail if no permissions, i think that is ok